### PR TITLE
add draco/1.3.5

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **draco/1.3.5**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
```
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets(312,5): error MSB3491: Could not write lines to file "draco_compression_attributes_pred_schemes_dec.dir\Debug\draco_co.517E10AA.tlog\draco_compression_attributes_pred_schemes_dec.lastbuildstate". The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters. [C:\J\c3ipr\pr_725_2_0_2\.conan\data\draco\1.3.5\_\_\build\8a53c449afc5d7e043474b43b2844541ed168c33\build_subfolder\source_subfolder\draco_compression_attributes_pred_schemes_dec.vcxproj]

C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets(312,5): error MSB3491: Could not write lines to file "draco_compression_attributes_pred_schemes_enc.dir\Debug\draco_co.83C85316.tlog\draco_compression_attributes_pred_schemes_enc.lastbuildstate". The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters. [C:\J\c3ipr\pr_725_2_0_2\.conan\data\draco\1.3.5\_\_\build\8a53c449afc5d7e043474b43b2844541ed168c33\build_subfolder\source_subfolder\draco_compression_attributes_pred_schemes_enc.vcxproj]
```

File: recipes/draco/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    description = "Draco is a library for compressing and decompressing 3D " \
+                  "geometric meshes and point clouds. It is intended to " \
+                  "improve the storage and transmission of 3D graphics."
+    license = "Apache-2.0"
+    topics = ("conan", "draco", "3d", "graphics", "mesh", "compression", "decompression")
+    homepage = "https://google.github.io/draco/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
```
Comment (uilianries) [inline recipes/draco/all/conanfile.py]:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
    short_paths = True
```
https://docs.conan.io/en/latest/reference/conanfile/attributes.html#short-paths

**Comment by madebr** [comment]
@uilianries
This package only needs short_paths when building, not when consuming the package.
Is it possible to instruct this to Conan?

**Comment by uilianries** [comment]
@madebr maybe by configure(self), but I'm not sure, only checking the code/behavior:

```python
def configure(self):
    self.short_path = self.develop
```

File: recipes/draco/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        del self.options.fPIC
```
Comment (madebr) [inline recipes/draco/all/conanfile.py]:
```suggestion
if self.settings.os == "Windows":
    del self.options.fPIC
```

File: recipes/draco/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+    def configure(self):
```
Comment (madebr) [inline recipes/draco/all/conanfile.py]:
```suggestion
    def configure(self):
        if self.options.shared:
            del self.options.shared
```

**Comment by madebr** [comment]
@uilianries 
> ```python
> def configure(self):
>     self.short_path = self.develop
> ```

I tried this but got the following error:
```
ERROR: The 'draco/1.3.5' package has 'exports_sources' but sources not found in local cache.
Probably it was installed from a remote that is no longer available.
```
This would be a nice feature for a next conan release, wouldn't it?
Essentially, only `source_folder` and `build_folder` should be "short_paths", `package_folder` can be normal.

**Comment by uilianries** [comment]
@madebr It's not possible using a condition for `short_paths` in Conanfile.py, I've checked now, both code and behavior. You can open a feature request if you are interested on it.

**Pull request merged**